assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.stories.js

```diff
@@ -192,6 +211,24 @@ TwoTiles.scenario = {
 	label: 'Modules/Analytics4/Components/AudienceSegmentation/Dashboard/AudienceTilesWidget/TwoTiles',
 };
 
+export const TwoTilesWithMissingCustomDimension = Template.bind( {} );
+TwoTilesWithMissingCustomDimension.storyName =
+	'TwoTilesWithMissingCustomDimension';
+TwoTilesWithMissingCustomDimension.args = {
+	configuredAudiences: [
+		'properties/12345/audiences/1', // All Users
+		'properties/12345/audiences/3', // New visitors
+	],
+	setupRegistry: ( registry ) => {
+		registry.dispatch( MODULES_ANALYTICS_4 ).setSettings( {
+			availableCustomDimensions: [],
+		} );
+	},
+};
+TwoTilesWithMissingCustomDimension.scenario = {
+	label: 'Modules/Analytics4/Components/AudienceSegmentation/Dashboard/AudienceTilesWidget/TwoTilesWithMissingCustomDimension',
+};
```

Comment:
I don't think we need the scenario for the two tile variant, after all there's no difference in how the missing custom dimension is handled for two vs three tiles.

It seems reasonable to keep the story, but I would drop the scenario (and remove its reference images).

```suggestion
```

Reply:
Removed.